src/main/webapp/instructorHelp.jsp

```diff
@@ -9108,185 +8898,25 @@
-                                                            </td>
-                                                        </tr>
-                                                        <tr>
-                                                            <td class="text-left">
-                                                                <div data-toggle="tooltip" data-placement="top" title="Control what instructors can view">
-                                                                    Instructors</div>
-                                                            </td>
-                                                            <td>
-                                                                <input class="visibilityCheckbox answerCheckbox" type="checkbox" value="INSTRUCTOR" checked="">
-                                                            </td>
-                                                            <td>
-                                                                <input class="visibilityCheckbox giverCheckbox" type="checkbox" value="INSTRUCTOR">
-                                                            </td>
-                                                            <td>
-                                                                <input class="visibilityCheckbox recipientCheckbox" type="checkbox" value="INSTRUCTOR">
-                                                            </td>
-                                                        </tr>
-                                                    </tbody>
-                                                </table>
-                                            </div>
-                                            <div class="form-group">
-                                                <textarea class="form-control" rows="3" placeholder="Your comment about this student" name="commenttext" id="commentText1">Comment from Instructor to Student Alice in kk.kk.-demo</textarea>
-                                            </div>
-                                            <div class="col-sm-offset-5">
-                                                <input id="commentsave-1" title="Save comment" onclick="return submitCommentForm('1');" type="submit" class="btn btn-primary" value="Save">
-                                                <input type="button" class="btn btn-default" value="Cancel" onclick="return disableComment('1');">
-                                            </div>
-                                        </div>
-                                    </form>
-                                </li>
-
-                            </ul>
-                        </div>
-                    </div>
                 </div>
             </li>
             <li>
                 <span class="text-bold">
                     <a name="searchComments">
-                        <h3>5. Search for comments</h3>
+                        <h3>3. Search for comments</h3>
```

Comment:
Not sure why I can't see the text in the code here, but "checkboxes" should probably be changed to "checkbox".
![checkboxes](https://cloud.githubusercontent.com/assets/7546744/26338221/ef22dd96-3fb0-11e7-8ffe-ded719eaaeaa.png)